client/manifest.json

```diff
@@ -3,6 +3,7 @@
   "name": "Echo Chamber",
   "description": "Echo Chamber Extension: Good information for everyone",
   "version": "1.0",
+  "permissions": ["storage"],
```

Comment:
what's this for?

Reply:
its something i saw to add while importing the MyChart library so i think it might be used for that. we can comment that out as of now